src/config/dynamic-theme-fixes.config

```diff
@@ -3665,6 +3665,13 @@ img.mega-menu-logo
 
 ================================
 
+cloudways.com
+
+INVERT
+.md-button.md-primary.md-raised
```

Comment:
Where can I find this element?

Reply:
I think there's one here: [Log in](https://platform.cloudways.com/login?ref_id=web_navbar), but definitely while you're logged in they are the main buttons. The font is a light purple color on a light green background and really hard to make out.